content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -138,10 +138,28 @@ In order to enable Preview deployment for pull requests, create a separate datab
 
 > **Note**: because each pull request shares the same preview database, if you have multiple pull requests with diverging schema migrations, you will encounter problems applying them.
 
+## <inlinecode>vercel/pkg</inlinecode> <inlinecode>ENOENT</inlinecode> error
+
+If you use [vercel/pkg](https://github.com/vercel/pkg) to package your Node.js project, then you might encounter an error like the following:
+
+```
+spawn /snapshot/enoent-problem/node_modules/.prisma/client/query-engine-debian-openssl-1.1.x ENOENT
+```
+
+To avoid this error, add your Prisma query engine binary path to the `pkg/assets` section of your `package.json` file, as follows:
+
+```json file=package.json copy
+{
+  "pkg": {
+    "assets": ["node_modules/.prisma/client/*.node"]
+  }
+}
+```
```

Comment:
What does this have to do with Vercel deployment?

`pkg` is a library that is created by the company Vercel. It has 0 connection with Vercel the deployment platform.

Reply:
As I understand it, `pkg` bundles your app into an exe to be used on Vercel. Isn't that a step in the task of deployment?

Reply:
No, Vercel can not even run executable/binaries. They run Node apps directly.

This is a general purpose tool (only coincidentally from Vercel) to package any Node app into an executable/binary. We use(d) it to package our Prisma CLI into an executable so Go users (for our Prisma Client Go) can use the CLI without having to install Node. Some of our users are doing similar things with their Node apps, so the users can just download 1 file and run their app locally.

Nothing to do with Vercel deployment at all. It would fit more into a "Weird usage and deployment options of Prisma" page, and could sit next to a section about "Using Prisma in Electron apps" for example. I don't think we have something like that yet though 😢